package.json

```diff
@@ -15,10 +15,12 @@
         "prelint:tsc": "yarn build",
         "lint:tsc": "lerna run lint:tsc --stream",
         "prebuild:storybook": "yarn build",
-        "build:storybook": "yarn workspace @vivid-planet/react-admin-stories build-storybook"
+        "build:storybook": "yarn workspace @vivid-planet/react-admin-stories build-storybook",
+        "intl:extract": "formatjs extract ./packages/**/*.ts* --out-file lang/de.json --ignore ./**.d.ts --ignore ./packages/react-admin-stories/**/* --format simple"
```

Comment:
`lang/en.json` instead of `lang/de.json` ? 
Default language is 'en" (in the end).

Reply:
Sounds good.